### DISCUSSION TITLE
auto denying my requests

### DISCUSSION THREAD

**Discussion by Lonely-Potato** [community/community#50952] [GitHub Education]
less then 1 min after my request is send i get a mail that my request is denied. I used multiple different documents to show that I'am a student. Right now it complains the most about my github profile not being complete even though i filled in almost every field correctly. I also want to use my student ID but our school didn't put a date on the ID.

**Comment by samus-aran** [comment]
Hi @Lonely-Potato  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
